### PR TITLE
Jmix guide: application events

### PR DESCRIPTION
**Pull request by mariodavid**

### CODE REVIEW

File: content/modules/guides/pages/decoupling-business-logic-with-application-events.adoc [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+1. xref:ROOT:setup.adoc[Setup Jmix Studio]
+2. Download the sample project. You can **download the completed sample project**, which includes all the examples used in this guide. This allows you to explore the finished implementation and experiment with the functionality right away.
+* https://github.com/jmix-framework/{sample-project}/archive/refs/heads/main.zip[Download^] and unzip the source repository
+* or clone it using git:
+`git clone https://github.com/jmix-framework/{sample-project}.git`
+
+Alternatively, you can **start with the base Petclinic project** and follow the step-by-step instructions in this guide to implement the features yourself: https://github.com/jmix-framework/jmix-petclinic-2[Jmix Petclinic] and follow along to add the functionality step-by-step.
+
+[[what-we-are-going-to-build]]
+== What We are Going to Build
```
Comment (knstvk) [inline content/modules/guides/pages/decoupling-business-logic-with-application-events.adoc:23]: What We Are Going to Build

**Review by knstvk — APPROVED** [review]
@mariodavid - we'll need to move the content to a different structure.

**Comment by knstvk** [comment]
See https://github.com/jmix-framework/jmix-docs/pull/879#issuecomment-2404153574

**Comment by mariodavid** [comment]
ok, done 👍 . see: https://github.com/jmix-framework/jmix-docs/pull/892

**Pull request closed without merging**